src/panels/lovelace/entity-rows/hui-media-player-entity-row.ts

```diff
@@ -22,14 +22,36 @@ import {
   SUPPORT_NEXT_TRACK,
   OFF_STATES,
   SUPPORT_PAUSE,
+  SUPPORT_TURN_ON,
+  SUPPORT_TURN_OFF,
+  SUPPORT_PREVIOUS_TRACK,
+  SUPPORT_VOLUME_SET,
+  SUPPORT_VOLUME_MUTE,
+  SUPPORT_VOLUME_BUTTONS,
 } from "../../../data/media-player";
 import { hasConfigOrEntityChanged } from "../common/has-changed";
+import { computeRTLDirection } from "../../../common/util/compute_rtl";
+import { debounce } from "../../../common/util/debounce";
 
 @customElement("hui-media-player-entity-row")
 class HuiMediaPlayerEntityRow extends LitElement implements LovelaceRow {
   @property() public hass?: HomeAssistant;
-
   @property() private _config?: EntityConfig;
+  @property() private _narrow?: boolean = false;
+  @property() private _veryNarrow?: boolean = false;
+  private _resizeObserver?: ResizeObserver;
+  private _debouncedResizeListener = debounce(
```

Comment:
Answered on Discord, but adding 
```css
:host {
  display: block
}
```

Fixes the issue here.

Reply:
That shouldn't impact parentElement ?

Reply:
it does indeed work